lib/engine/game/g_18_cz.rb

```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+require_relative '../config/game/g_18_cz'
+require_relative 'base'
+
+module Engine
+  module Game
+    class G18CZ < Base
+      register_colors(brightGreen: '#c2ce33', beige: '#e5d19e', lightBlue: '#1EA2D6', mintGreen: '#B1CEC7',
+                      yellow: '#ffe600', lightRed: '#F3B1B3')
+
+      load_from_json(Config::Game::G18CZ::JSON)
+
+      DEV_STAGE = :prealpha
```

Comment:
:prealpha is default value so not needed.

Reply:
I mean it will be added once it is in alpha, so why should I remove it?